cmake/FindQGIS.cmake

```
@@ -122,13 +122,12 @@ if(QGIS_FOUND)
   set(QGIS_INCLUDE_DIRS ${QGIS_INCLUDE_DIR})
 endif()
 
-find_package(Qca REQUIRED)
-target_link_libraries(QGIS::Core INTERFACE qca)
-find_package(Qt5Keychain REQUIRED)
-target_link_libraries(QGIS::Core INTERFACE Qt5Keychain::Qt5Keychain)
-
 if(WITH_VCPKG)
   include("cmake/qgis-cmake-wrapper.cmake")
 else()
   target_include_directories(QGIS::Core INTERFACE ${GDAL_INCLUDE_DIR})
+  find_package(QCA REQUIRED)
```

```suggestion
  find_package(Qca-qt5 REQUIRED)
```

and remove the `FindQCA.cmake` file again, that's hopefully good enough for now